presto-common/src/main/java/com/facebook/presto/common/clustering/Distribution.java

```diff
@@ -0,0 +1,40 @@
+package com.facebook.presto.common.clustering;
+
+import com.facebook.presto.common.type.Type;
+import javafx.util.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// Assume: We have 100 values to represent 100 percentiles
+// for the column for now.
+public final class Distribution
```

Comment:
Let's try to reuse existing helpers like `Range` or `Block`. I think in this case, we could have a simple Block to represent values in sorted order.

Reply:
Yeah, after we finalize the logic, I will refactor these classes.